README.md

```diff
@@ -196,6 +196,10 @@ content: ({tiastId, dismiss}) => (
 
 ![alt tag](https://raw.githubusercontent.com/ealush/butter-toast/master/assets/rec0.gif)
 
+# Universal app build
+To use ButterToast in universal app you should run `npm run build:universal`
+and import it from `butter-toast/dist/universal`.
```

Comment:
I'd add a more detailed example on how to actually consume the package in server side rendering:
```js
import ButterToast from 'butter-toast/dist/universal/index.js';
import 'butter-toast/dist/universal/style.css';
```

Reply:
Yeah make sense, what about that:

# Universal / Server side rendered apps
ButterToast uses webpack's `style-loader` which injects CSS into JS module and depends on `window` object - thus server-side rendering would raise an error.
To bypass that you can consume the universal build, which outputs the style to CSS file instead JS:
```
import ButterToast from 'butter-toast/dist/universal/index.js';
import 'butter-toast/dist/universal/style.css';
```

Reply:
Much better. Use it :)